src/connectors/hits/__tests__/connectHitsWithInsights-test.js

```diff
@@ -0,0 +1,46 @@
+import jsHelper, { SearchResults } from 'algoliasearch-helper';
+import connectHitsWithInsights from '../connectHitsWithInsights';
+
+jest.mock('../../../lib/utils/hits-absolute-position', () => ({
+  addAbsolutePosition: hits => hits,
+}));
+
+
+describe('connectHitsWithInsights', () => {
+  it('should expose `insights` props', () => {
+    const rendering = jest.fn();
+    const makeWidget = connectHitsWithInsights(rendering);
+    const widget = makeWidget();
+
+    const helper = jsHelper({}, '', {});
+    helper.search = jest.fn();
+
+    widget.init({
+      helper,
+      state: helper.state,
+      createURL: () => '#',
+      onHistoryChange: () => {},
+      instantSearchInstance: {
+        insightsClient: jest.fn(),
+      },
+    });
+
+    const firstRenderingOptions = rendering.mock.calls[0][0];
+    expect(typeof firstRenderingOptions.insights).toBe('function');
```

Comment:
You can use [`toBeInstanceOf`](https://jestjs.io/docs/en/expect#tobeinstanceofclass) for better error messages.

Reply:
fixed here 9a005ab